src/backend/distributed/executor/multi_utility.c

```diff
@@ -87,6 +91,10 @@
 #include "utils/syscache.h"
 
 
+#define LOCK_RELATION_IF_EXISTS "SELECT lock_relation_if_exists('%s', '%s');"
+#define REMOTE_LOCK_MODE_FOR_TRUNCATE "ACCESS EXCLUSIVE"
```

Comment:
I don't have a good reason/explanation for this, but,  `REMOTE_LOCK_MODE_FOR_TRUNCATE ` feels like a better fit for the function definition, with `char *remoteLockModeForTruncate = "ACCESS EXCLUSIVE"`. I think that is because we don't have any other macros that defines a lock mode.

Feel free to ignore.

Reply:
gone after ditching lock mode text to get it from enum.